.github/scripts/label_promoted_commits.py

```diff
@@ -23,36 +23,91 @@ def parser():
                              'commit, exclusive).')
     parser.add_argument('--update_issue', type=bool, default=False, help='Set True to update issues when backport was '
                                                                          'done')
-    parser.add_argument('--label', type=str, required=True, help='Label to use')
+    parser.add_argument('--label', type=str, default='promoted-to-master', help='Label to use')
+    parser.add_argument('--ref', type=str, required=True, help='PR target branch')
     return parser.parse_args()
 
 
+def add_comment_and_close_pr(pr_number, repository, headers, comment):
```

Comment:
alternatively, i'd suggest use the `github` python module like:
```py
def add_comment_and_close_pr(pr_number, repository, headers, comment):
    issue = repository.get_issue(pr_number)
    issue.create_comment(comment)
    pull = repository.get_pull(pr_number)
    pr.edit(state="closed", body=comment)
```

> Today after Mergify opened a Backport PR, it will stay open until someone manually closes it

and, i am not what "it" implies. is it an issue or a pull request? if what you really want is to comment on a pull request, as the commit message puts:

> Add a comment on the backport PR and close it

probably we should instead use something like:
```py
def add_comment_and_close_pr(pr_number, repository, headers, comment):
    pull = repository.get_pull(pr_number)
    pull.create_comment(comment)
    pr.edit(state="closed", body=comment)
```
instead.

Reply:
@tchaikov Thanks for your suggestion, implemented it with a small change using `pr.create_issue_comment` instead

Also updated the PR description to make it more clear

Reply:
@yaronkaikov it seems we can close an PR *with* a comment, like
```py
pr.edit(state="closed", body=comment)
```

is this acceptable ? IMHO, it's even better. what do you think?